docs/api/ipc-renderer.md

```diff
@@ -1,29 +1,41 @@
 # ipc (renderer)
 
 The `ipc` module provides a few methods so you can send synchronous and
-asynchronous messages to the main process, and also receive messages sent from
-main process. If you want to make use of modules of main process from renderer
+asynchronous messages from the render process (web page) to the main process. You can also receive messages returned from the main process.
+
+**Note**: If you want to make use of modules in the main process from the renderer
 process, you might consider using the [remote](remote.md) module.
 
-See [ipc (main process)](ipc-main-process.md) for examples.
+See [ipc (main process)](ipc-main-process.md) for code examples.
+
+## Methods
+
+The `ipc` module has the following methods for sending messages:
+
+**Note**: When using these methods to send a `message` you must also listen for it in the main process with [`ipc (main process)`](ipc-main-process.md).
+
+### `ipc.send(channel[, message])`
```

Comment:
Actually `ipc.send` can send arbitrary numbers of arguments, like `ipc.send('channel', 1, 2, 3, 4, 5)`, which behaves the same with `EventEmitter.emit`, maybe we can follow [Node's conventions](https://nodejs.org/api/events.html#events_emitter_emit_event_arg1_arg2) here by using `(channel[, arg1][, arg2][, ...])`.